src/main/scala/com/gu/invoicing/refund/Impl.scala

```diff
@@ -120,7 +120,7 @@ object Impl {
       val chargeAmountToRefund = invoiceItem.ChargeAmount.min(amountToRefund)
       val chargeAdjustment = List(
         InvoiceItemAdjustmentWrite(
-          invoiceItem.ChargeDate.toLocalDate,
+          LocalDate.now(),
```

Comment:
This will reintroduce the bug which the other PR was trying to fix won't it?

Reply:
Yes it would!

Could we find a resolution to the original bug that allows us to set the `AdjustmentDate` to the current date and avoid the timezone discrepancy? 

Jessica suggested there might be a per-user timezone setting in Zuora, and if there is then we could set this to UTC. Then both server and Zuora should be set to the same time and avoid the original bug.

I might be misunderstanding the original bug, but isn't it quite an extreme edge-case where the invoice/payment and refund are all on the same day in the hour after midnight during BST? In any other case, even if the adjustment date is incorrectly evaluated to be a day earlier than now (yesterday), the invoice date would be still be yesterday or earlier. I definitely might be misunderstanding though.